lms/static/sass/features/_course-experience.scss

```diff
@@ -171,3 +171,21 @@
     }
   }
 }
+
+// Course Updates Page
+div.course-updates {
+  div.all-updates {
+      article.updates-article {
+        margin: 10px 0px;
+        .date {
+          font-size: 14px;
+          font-weight: 300;
+        }
+        .end-line {
+          background-color: #eee;
```

Comment:
For theming again, we don't hard-code colors. I'd use:

    background-color: $lms-border-color;